index.html

```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
         <code>alg</code>, <code>iat</code>,
         <code>exp</code> and <code>cnf</code>.
       </p>
+      <p>
+        When <code>exp</code> or <code>exp</code> JWT claims are present,
+        they represent issuance and expiration time of a signature, respectively.
```

Comment:
they represent the issuance and expiration times of the signature, respectively.

Reply:
```suggestion
        they represent the issuance and expiration time of the signature, respectively.
```